ts/components/OrganizationHeader.tsx

```diff
@@ -20,6 +20,8 @@ type Props = {
   service: ServicePublic;
 };
 
+const leftColumnStyle = { marginRight: 16 };
```

Comment:
We already have a style variable in this component, is it needed to add one more const instead of extending style object?

https://github.com/pagopa/io-app/blob/11cd0a81e4a70d78b889bda35dbb2654619b7d74/ts/components/OrganizationHeader.tsx#L9